### ISSUE TITLE
Cannot get integration successfully set up

### ISSUE THREAD

**Issue by BewilderedApe** [home-assistant/home-assistant.github.io#34408]
### Feedback

I've meticulously attempted to follow the wiki three times. Same end result each time.

When I get to the last step of linking the Google account it fails. I'm not sure how to even troubleshoot, or where I'm going wrong.

<img width="2557" alt="Screenshot 2024-08-22 at 3 34 21 PM" src="https://github.com/user-attachments/assets/0c0f1c82-b3ff-4763-aa70-1094afdc067e">

I am a novice. I'll keep searching for help, but any guidance would be greatly appreciated and will hopefully help the community. 

### URL

https://www.home-assistant.io/integrations/nest/

### Version

2024.8.2

**Comment by allenporter** [comment]
Check out the troubleshooting section https://www.home-assistant.io/integrations/nest/#troubleshooting under "Can’t link to" and click the area under "Resolving mismatched OAuth Client ID" which has instructions how to repair.

**Comment by BewilderedApe** [comment]
Thanks, Allen. I figured out my issue. Sharing here in case it's helpful to add somewhere in the documentation.

I had to delete the Google Nest OAuth client ID from my first attempts by going to Home Assistant → **Settings** → **Devices & services** → **Application credentials** (3 dots in the top right of screen)

This completely restarted the Home Assistance setup process and allowed me to complete the integration.

----

I have a new problem, however. 

I am trying to integrate the feed into Apple Home and I cannot get the live feed to show. I've figured out how to have the live feed show on Home Assistant, but not on Home. I'm in the early stages of researching, but if you have any advice or references it would be much appreciated.
